Makefile

```diff
@@ -0,0 +1,177 @@
+# Orchestrator
+ORCHESTRATOR_PATH := $(shell pwd)
+
+# Bootstrapper
+OPERATOR_ADDRESS := 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+BOOTSTRAP_OUTPUT_PATH := $(shell pwd)/build/bootstrap.json
+BOOTSTRAPPER_COMMIT := b0b647500c2ae3e3b0d99e345fa652989bca4726
+BOOTSTRAPPER_PATH := $(shell pwd)/madara-bootstrapper
+VERIFIER_ADDRESS := 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+
+# SNOS
+CAIRO_LANG_COMMIT := a86e92bfde9c171c0856d7b46580c66e004922f3
+
+# Madara
+MADARA_PATH := $(shell pwd)/madara
+MADARA_COMMIT := ad0ec8cdfcde03a8e2feef76d64a7fba7a9fb792
+MADARA_DATA_PATH := $(shell pwd)/build/madara
+
+# Pathfinder
+PATHFINDER_COMMIT := 138140e5fd967ede92806fd62de47c2e6b65712a
+PATHFINDER_DATA_PATH := $(shell pwd)/build/pathfinder
+PATHFINDER_PATH := $(shell pwd)/pathfinder
+
+# Environment file
+ENV_FILE := $(shell pwd)/.makefile.json
+PID_FILE := $(shell pwd)/.pids.json
+
+.PHONY: all anvil madara core-contract update-madara eth-bridge pathfinder orchestrator cleanup
+
+define save_pid
+	if [ ! -f "$(PID_FILE)" ]; then echo '{}' > $(PID_FILE); fi && \
+	jq --arg key "$(1)" --arg value "$(2)" '. + {($key): ($value)}' $(PID_FILE) > $(PID_FILE).tmp && \
+	mv $(PID_FILE).tmp $(PID_FILE)
+endef
+
+define get_pid
+	jq -r '.$(1) // empty' $(PID_FILE)
+endef
+
+setup:
```

Comment:
getting this error


```
Starting setup process...
Cleaning up processes...
Cleanup completed.
Cleaning up processes...
Cleanup completed.
jq: error: ey/0 is not defined at <top-level>, line 1:
. + {(ey): (alue)}
jq: error: alue/0 is not defined at <top-level>, line 1:
. + {(ey): (alue)}
jq: 2 compile errors
make: *** [setup] Error 3
anvil --block-time 6
```

Reply:
this is a shell error when used with bash it resolves. I have added the changes so that this issue doesn't pop up regardless of the terminal used